cmd/osio/cmd/token.go

```diff
@@ -0,0 +1,46 @@
+package cmd
+
+import (
+	"github.com/spf13/cobra"
+
+	log "github.com/sirupsen/logrus"
+
+	"fmt"
+	"github.com/fabric8-services/fabric8-jenkins-proxy/internal/util"
+	"net/url"
+)
+
+var (
+	cmdToken = &cobra.Command{
+		Use:   "token",
+		Short: "Prints the JSON access token for a user.",
+		Long:  `Prints the OpenShift IO JSON (JWT) accesss token for the provided Red Hat Developers credentials.`,
+		Run:   runCreateToken,
+	}
+	targetEnv string
+	username  string
+	password  string
+	encode  bool
+)
+
+func init() {
+	cmdToken.Flags().StringVarP(&targetEnv, "target", "t", "stage", "Target environment OpenShift.io stage vs prod.")
+	cmdToken.Flags().StringVarP(&username, "username", "u", "", "Red Hat Developer username.")
+	cmdToken.Flags().StringVarP(&password, "password", "p", "", "Red Hat Developer password.")
+	cmdToken.Flags().BoolVarP(&encode, "encode", "e", false, "Whether or not the output should be URL encoded.")
+}
+
+func runCreateToken(cmd *cobra.Command, args []string) {
```

Comment:
@alexeykazakov, FYI, this is the helper tool I ended up creating as part of _fabric8-jenkins-proxy_. Given Red Hat Developers credentials it spits out the JSON access token. I would like to extend this to also create OpenShift access token and maybe some other things. My idea would be that this tool can help in setting up scripted access to the system, for example for testing purposes. 

Benefit over the offline token is that this does not expire and I don't have any manual steps. That said, osio could also be used to create the offline token. Same same really.